src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -7,180 +7,128 @@
     list-style-type: none;
     vertical-align: top;
     margin: 0;
     padding: 0;
     display: inline-block;
+    position: relative;
 
     &:last-of-type {
       .ms-Breadcrumb-chevron {
         display: none;
       }
     }
-  }
-}
-
-.ms-Breadcrumb-overflow {
-  display: none;
-  position: relative;
 
-  .ms-Breadcrumb-overflowButton {
-    font-size: 16px;
-    display: inline-block;
-    color: $ms-color-themePrimary;
-    padding: 8px;
-    cursor: pointer;
-    vertical-align: top;
-  }
-}
-
-.ms-Breadcrumb-overflowMenu {
-  display: none;
-  position: absolute;
-  margin-right: -2px;
-
-  &.is-open {
-    display: block;
-    top: 36px;
-    left: 0;
-    @include drop-shadow;
-    background-color: $ms-color-white;
-    border: 1px solid $ms-color-neutralTertiaryAlt;
-    z-index: ($ms-zIndex-ContextualMenu + $ms-zIndex-middle);
-  }
-
-  &::before {
-    position: absolute;
-    @include drop-shadow;
-    top: -6px;
-    left: 6px;
-    content: ' ';
-    width: 16px;
-    height: 16px;
-    transform: rotate(45deg);
-    background-color: $ms-color-white;
+    .ms-Breadcrumb-itemLink {
+      @include focus-border;
+      @include ms-font-xl;
+      display: inline-block;
+      padding: 0 4px;
+      max-width: $Breadcrumb-itemMaxWidth;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      overflow: hidden;
+      vertical-align: top;
+    }
   }
 
-  .ms-ContextualMenu {
-    border: 0;
-    box-shadow: none;
+  .ms-Breadcrumb-overflow {
+    display: inline-block;
     position: relative;
-    width: 190px;
 
-    &.is-open {
-      margin-bottom: 0;
+    .ms-Breadcrumb-overflowButton {
+      font-size: $Breadcrumb-overflowButtonSize;
+      display: inline-block;
+      color: $Breadcrumb-overflowButtonColor;
+      padding: 8px;
+      cursor: pointer;
+      vertical-align: top;
     }
   }
-}
-
-.ms-Breadcrumb-itemLink,
-.ms-Breadcrumb-overflowButton {
-  text-decoration: none;
-  outline: transparent;
-
-  &:hover {
-    background-color: $ms-color-neutralLighter;
-    cursor: pointer;
-  }
-
-  &:focus {
-    outline: $ms-color-neutralSecondaryAlt solid 1px;
-    color: $ms-color-black;
-  }
 
-  &:active {
+  .ms-Breadcrumb-itemLink,
+  .ms-Breadcrumb-overflowButton {
+    text-decoration: none;
     outline: transparent;
-    background-color: $ms-color-neutralTertiaryAlt;
-  }
-}
-
-.ms-Breadcrumb-itemLink {
-  font-weight: $ms-font-weight-light;
-  font-size: $ms-font-size-xl;
-  color: $ms-color-neutralPrimary;
-  display: inline-block;
-  padding: 0 4px;
-  max-width: 160px;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-  overflow: hidden;
-  vertical-align: top;
-}
 
+    &:hover {
+      background-color: $selectedHoverBackgroundColor;
+      color: $Breadcrumb-buttonHoverColor;
+      cursor: pointer;
+    }
 
-@media screen and (max-width: $ms-screen-md-max) {å
-  .ms-Breadcrumb {
-    margin: 10px 0;
-  }
+    &:focus {
+      outline: $ms-color-neutralSecondaryAlt solid 1px;
```

Comment:
nit: typically this would be in the order `1px solid $ms-color-neutralSecondaryAlt`